### PR TITLE
Installation without docker

### PR DESCRIPTION
**Pull request by JustasBelevicius**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,49 +25,51 @@ Development endpoint: https://petbook-back-dev.herokuapp.com/graphql
...
-    local               pets-back_database-data
-```
-
-3. Delete the volume `docker volume rm pets-back_database-data`
-4. Run `docker-compose up` to start the stack
-5. Check if the are no SQL errors
+## B1. Setup with docker
+
+1. Install npm libraries with `npm install` command (if not installed previously).
+2. Copy and rename `database.env.sample` to `database.env`. `POSTGRES_USER`, `POSTGRES_PASSWORD`, `POSTGRES_DB` can be adjusted to your liking, but the default configuration will work as well.
```
Comment (AzisK) [inline README.md:38]: Very good

File: README.md [hunk 2]
```diff
@@ -23,49 +25,51 @@ Development endpoint: https://petbook-back-dev.herokuapp.com/graphql
...
+4. Make sure Docker is running.
+5. Run `docker-compose up -d`
+
+## B2. Setup without docker
+
+1. Download the latest supported version of PostgreSQL https://www.postgresql.org/download/
+> e.g. Windows 7 and 8 support PostgreSQL version 10
+2. Install PostgreSQL, take note of the password and port you've chosen during the installation.
+> You might have to restart your PC after the installation is complete
+3. Start pgAdmin and connect to your server using the password you've chosen (should be visible under Servers dropdown on the left)
```
Comment (AzisK) [inline README.md:49]: I believe the user might not have pgAdmin, so maybe a step before would be to download it

File: README.md [hunk 3]
```diff
@@ -23,49 +25,51 @@ Development endpoint: https://petbook-back-dev.herokuapp.com/graphql
...
+5. Create a new database:
+    - Right click "Databases" and select Create > Database.
+    - Under the "General" tab, set the Name field (e.g. "gis"). Take note of the value you have chosen.
+    - Click "Save".
+6. Upload data to tables:
+    - Right click on the database entry you've just created and select "Query Tool"
+    - Grant privileges to your new user: `GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO username_from_step_4`;
+    - For each file inside `/database` folder:
+        - Click on the little folder icon inside the query tool
+        - Choose the file (e.g. "1-schema.sql") and click Select
```
Comment (AzisK) [inline README.md:65]: Maybe we should mention that they should be executed in order: 1,2,3. At least `1-schema.sql` has to go first for sure

File: package.json [hunk 4]
```diff
@@ -12,12 +12,13 @@
   "scripts": {
     "build": "tsc",
     "start": "node dist/src/index.js",
+    "win-dev": "tsc && del combined.env && type database.env >> combined.env && echo. >> combined.env && type common.env >> combined.env && nodemon -r dotenv/config dist/src/index.js dotenv_config_path=combined.env",
```
Comment (AzisK) [inline package.json:15]: What does `echo.` do?

**Review by AzisK — APPROVED** [review]
Good job! Left a few notes

File: package.json (hunk 4 above)
Comment (JustasBelevicius) [reply package.json:15]: Echos a new line. windows specific.

**Pull request merged**